crates/pop-parachains/src/up/mod.rs

```diff
@@ -1123,6 +1151,77 @@ default_command = "./target/release/parachain-template-node"
 			Ok(())
 		}
 
+		#[tokio::test]
+		async fn new_with_local_parachain_without_path_works() -> Result<()> {
+			let temp_dir = tempdir()?;
+			let cache = PathBuf::from(temp_dir.path());
+			let config = Builder::new().suffix(".toml").tempfile()?;
+			writeln!(
+				config.as_file(),
+				r#"
+[relaychain]
+chain = "rococo-local"
+
+[[parachains]]
+id = 1000
+
+[parachains.collator]
+name = "collator"
+command = "parachain-template-node"
+
+[[parachains]]
+id = 2000
+
+[parachains.collator]
+name = "collator"
+command = "substrate-contracts-node"
+"#
+			)?;
+			assert!(matches!(
```

Comment:
Would be great to indicate that we are expecting this to fail because having not provided a custom path, the binaries are expected in the default build dir.